docs/mesos/RpmCreateInstallMesos.md

```diff
@@ -0,0 +1,47 @@
+# Create and Install the Mesos + Net-Modules RPM
+This guide will walk you through building and installing Mesos + Net-Modules from RPMs. These RPMs will automate the installation of everything needed to run Mesos + Net-Modules in your cluster.
+
+## Build and Install the RPM
+The Mesos + Net-Modules RPMs can be built from the [net-modules repository](https://github.com/mesosphere/net-modules) by calling `make builder-rpm`. Once the RPMs are created, install the packages.
```

Comment:
Lets simplify this, the commands speak for themselves:

The Mesos + Net-Modules RPMs can be easily built and installed via the net-modules repository: